Penultimate tweaks for merged dashboard go live

Bjwebb opened the pull request:
[Merged dashboard go live milestone](https://github.com/IATI/IATI-Dashboard/milestone/33)

Bjwebb committed:
element: Fix broken links for @xml:lang

https://github.com/IATI/IATI-Dashboard/issues/673

Bjwebb committed:
element: Temporarily remove incorrect columns

https://github.com/IATI/IATI-Dashboard/issues/674

Bjwebb committed:
summary_stats: Use frequency calculation in the summary stats correctly

Bjwebb committed:
fix: Always order publishers by human_readable_name

https://github.com/IATI/IATI-Dashboard/issues/664

Bjwebb committed:
publisher: Remove links from publisher codelist counts

https://github.com/IATI/IATI-Dashboard/issues/653

chrisarridge reviewed (commented):
The flag colours were also changed in forward looking, so the text will also need to be changed there as well as in the timeliness templates.  Otherwise looks fine to me.

Bjwebb committed:
timeliness, forwardlooking: Use bs red/yellow colours for flags

Longer term, we should stop communicating solely with colour see:
https://github.com/IATI/IATI-Dashboard/issues/665
